metadata_tools/geometry_support.py

```
@@ -177,40 +191,231 @@ def __init__(self, observation, volume_id, meshgrids, level):
+            inventory = self.observation.inventory(bodies, expand=config.EXPAND, cache=False)
+            return inventory
+
+        # A RuntimeError is probably caused by missing spice data. There is
+        # probably nothing we can do.
+        except (OSError, RuntimeError) as e:
+            error = str(e)
+
+            # If no C-kernel data for this observation, proceed with a warning and set the 
+            # pointing_available flag.
+            if 'SPICE(NOFRAMECONNECT)' in error or 'SPICE(CKINSUFFDATA)' in error:
+                logger.warn(str(e))
```

Ruff (0.14.10)

Logging statement uses `warn` instead of `warning`

Convert to `warning`

(G010)